src/sentry/models/release.py

```diff
@@ -103,9 +104,58 @@ def to_string(cls, value):
             raise ValueError(repr(value))
 
 
-class ReleaseQuerySet(models.QuerySet):
-    SEMVER_FAKE_PACKAGE = "__sentry_fake__"
+@dataclass
+class SemverFilter:
+    operator: str
+    version_parts: Sequence[Union[int, str]]
+    package: Optional[str] = None
+
+
+SEMVER_FAKE_PACKAGE = "__sentry_fake__"
+SEMVER_WILDCARDS = frozenset(["X", "*"])
```

Comment:
To confirm my understanding of this a bit, `1.5.0.X` isn't valid semver, so there isn't any chance we'll accidentally wildcard what is a valid patch version right?

Reply:
Yep, that's right. I'm only supporting wildcards in the numeric portion of the version, so there should be no conflict